tools/docker/builder/rdkb/toolchain/bitbake.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (c) 2020 Tomer Eliyahu (Intel)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+scriptdir="$(cd "${0%/*}"; pwd)"
+rootdir="${scriptdir%/*/*/*/*/*}"
+
+. ${rootdir}/tools/docker/functions.sh
+
+usage() {
+    echo "usage: $(basename $0) [-hrv]"
+    echo "  options:"
+    echo "      -h|--help - show this help menu"
+    echo "      -r|--rdkb-root - path to rdkb root (default is PWD)"
+    echo "      -i|--build-image - Only build image"
+    echo "      -v|--verbose - verbosity on"
+    echo ""
+}
+
+build_image() {
+    docker build --tag rdkb-builder "$scriptdir/"
+}
+
+main() {
+
+    OPTS=`getopt -o 'hir:v' --long help,image,rdkb-path:,verbose -n 'parse-options' -- "$@"`
+
+    if [ $? != 0 ] ; then err "Failed parsing options." >&2 ; usage; exit 1 ; fi
+
+    eval set -- "$OPTS"
+
+    while true; do
+        case "$1" in
+            -h | --help)               usage; exit 0; shift ;;
+            -i | --image)              build_image; exit 0; shift ;;
+            -r | --rdkb-root)          RDKB_ROOT="$2"; shift; shift ;;
```

Comment:
`shift` accepts an argument (the number of args to shift), you can use `shift 2` instead of `shift; shift`